### PR TITLE
Feat/update and cleanup

### PR DESCRIPTION
**Pull request by 50bhan**

### CODE REVIEW

**Comment by 50bhan** [comment]
@mohammad-fouladgar Can you please remove the `scrutinizer` hook? It keeps failing.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 [![Build Status](https://travis-ci.org/mohammad-fouladgar/eloquent-builder.svg?branch=master)](https://travis-ci.org/mohammad-fouladgar/eloquent-builder)
 [![Coverage Status](https://coveralls.io/repos/github/mohammad-fouladgar/eloquent-builder/badge.svg?branch=develop)](https://coveralls.io/github/mohammad-fouladgar/eloquent-builder?branch=develop)
-[![Quality Score](https://img.shields.io/scrutinizer/g/mohammad-fouladgar/eloquent-builder.svg?style=flat-square)](https://scrutinizer-ci.com/g/mohammad-fouladgar/eloquent-builder)
 [![StyleCI](https://github.styleci.io/repos/144369188/shield?branch=develop)](https://github.styleci.io/repos/144369188)
```
Comment (mohammad-fouladgar) [inline README.md:7]: I think we no longer need this badge, right? We can use GitHub action for this.

**Comment by mohammad-fouladgar** [comment]
@50bhan  I disabled the `scrutinizer` hook.

File: README.md (hunk 1 above)
Comment (50bhan) [reply README.md:7]: Yes, that’s why I removed the badge.

**Comment by 50bhan** [comment]
@mohammad-fouladgar Great! It's working right now! Thanks.

File: README.md (hunk 1 above)
Comment (mohammad-fouladgar) [reply README.md:7]: I mean the StyleCI badge.

Comment (50bhan) [reply README.md:7]: @mohammad-fouladgar Hah! Yes, you're right! So, please as you're bumping a version up, remove this one as well.

**Pull request merged**